openmc/mgxs_library.py

```diff
@@ -0,0 +1,716 @@
+from collections import Iterable
+from numbers import Real, Integral
+from xml.etree import ElementTree as ET
+import warnings
+import sys
+if sys.version_info[0] >= 3:
+    basestring = str
+
+import numpy as np
+
+import openmc
+from openmc.mgxs import EnergyGroups
+from openmc.checkvalue import check_type, check_value, check_greater_than, \
+                              check_iterable_type
+from openmc.clean_xml import *
+
+# Supported incoming particle MGXS angular treatment representations
+REPRESENTATIONS = ['isotropic', 'angle']
```

Comment:
I would make this private, i.e., `_REPRESENTATIONS`, unless you plan on using it in other modules.